NEWS.md

```
@@ -1,3 +1,57 @@
+# **cfbfastR (development version)**
+
+### Expected Points model now comes from the shared `cfb_model_artifacts` bundle
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)